certbot-apache/certbot_apache/parser.py

```diff
@@ -28,7 +28,7 @@ class ApacheParser(object):
     arg_var_interpreter = re.compile(r"\$\{[^ \}]*}")
     fnmatch_chars = set(["*", "?", "\\", "[", "]"])
 
-    def __init__(self, aug, root, vhostroot, version=(2, 4)):
+    def __init__(self, aug, root, vhostroot, version=(2, 4), extra_path=False):
```

Comment:
nit: Since `vhostroot` is only used now if `extra_path` is `True`, can we just make `vhostroot` an optional parameter and remove `extra_path`?